app/controllers/main/routes.py

```diff
@@ -116,7 +116,29 @@ def RemoveRSVP():
     currentRsvpParticipant.delete_instance()
 
     flash("Successfully unregistered for event!", "success")
-    return redirect(url_for("admin.editEvent", eventId=event.id))
+    return redirect(url_for("admin.editEvent", eventId=eventId, program=program))
+
+@main_bp.route('/<username>/serviceTranscript', methods = ['GET'])
+def serviceTranscript(username):
+    if username != g.current_user and not g.current_user.isAdmin:
```

Comment:
`g.current_user` is a user object and `username` is a text string. You should compare like with like.